api/opencl_platform_layer.asciidoc

```diff
@@ -1483,6 +1483,13 @@ include::{generated}/api/version-notes/CL_DEVICE_PREFERRED_WORK_GROUP_SIZE_MULTI
         (Refer also to {clGetKernelWorkGroupInfo} where {CL_KERNEL_PREFERRED_WORK_GROUP_SIZE_MULTIPLE}
         can return a different value to {CL_DEVICE_PREFERRED_WORK_GROUP_SIZE_MULTIPLE} which may be more optimal.)
 
+| {CL_DEVICE_LATEST_CTS_VERSION_TESTED_anchor}
+
+include::{generated}/api/version-notes/CL_DEVICE_LATEST_CTS_VERSION_TESTED.asciidoc[]
+  | char[]
+      | Returns the latest version of the conformance test suite that this device
+        has been tested against.
```

Comment:
Maybe a little pedantic, but should we make it explicit that the device needs to have passed all of the tests in the version reported here? The phrase "tested against" doesn't necessarily imply this.

Reply:
Yes, I also thought we should ultimately reference the exact process here but it doesn't exist yet. I could say "successfully tested" but this could still be interpreted as "the tests ran end to end". I'll try to find something that's reasonably unambiguous.